CONTRIBUTING.md

```diff
@@ -47,7 +47,19 @@ Flag                  | Effect
 ./run-tests -k        | Run all tests even if some tests are failing
 ./run-tests -o gtest  | The gtest output format might be easier to read if you are using print statements for debugging.
 
-For convenience, when the test script is run without any parameters, it also runs the linter at the end.
+For convenience, when the test script is run without any busted parameters, it also runs the linter at the end.
+
+#### Testing garbage collection
+
+Doing proper GC testing takes longer, so we have a special mode for doing that.
+For testing garbage collection more carefully, pass "gc" as the first parameter
+of the script. The rest of the parameters will do the same as before:
+
+```sh
+$ ./run-tests gc                       # Run all tests but with proper GC testing
+$ ./run-tests gc spec/parser_spec.lua  # Run just one of the test suite files
+                                       # but with proper GC testing
+```
```

Comment:
What if instead of a magic positional argument, we used an environment variables? We could make a new one just for GC

    HARDMEMTESTS=1 ./runtests

or we could fit it with CFLAGS. (This would also work for other things that play with cflags, such as -fsanitize)

    EXTRACFLAGS=-DHARDMEMTESTS ./runtests

I'm not a fan of the magic positional argument, because it would make more sense to be a --flag instead. And now that I think about it, I also don't like a --flag because we pass all flags through to busted. It would be strange to have only a single flag that's different.

Reply:
I like the EXTRACFLAGS idea, it's even more general. Will do that.